product_multi_price/migrations/11.0.1.2.0/post-migration.py

```diff
@@ -0,0 +1,9 @@
+#  Copyright 2020 Tecnativa - Ernesto Tejeda
```

Comment:
You have to put a higher version than the existing one, and bump manually the module version, so there's no possibility of this not being triggered, even if ocabot is launched with other bump. Use `11.0.2.0.0`, as this is a major change in the module.